### PR TITLE
test out 3.5.0-alpha

### PR DESCRIPTION
**Pull request by tijcolem**
DO NOT MERGE. Testing against 3.5.0-rc1.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //Jenkins pipelines are stored in shared libraries. Please see: https://github.com/NREL/cbci_jenkins_libs
  
-@Library('cbci_shared_libs') _
+@Library('cbci_shared_libs@3.5.0-alpha') _
```
Comment (jmarrec) [inline Jenkinsfile:3]: * [ ] TODO: revert

File: lib/measures/SetCOPforSingleSpeedDXCoolingUnits/measure.rb [hunk 2]
```diff
@@ -268,8 +268,8 @@ def get_total_costs_for_objects(objects)
         if !hVACComponent.empty?
           hVACComponent = hVACComponent.get
 
-          # change and report high speed cop
-          initial_cop = hVACComponent.ratedCOP
+          # change and report high speed cop. Prior to 3.5.0 it was an OptionalDouble
+          initial_cop = OpenStudio::OptionalDouble.new(hVACComponent.ratedCOP)
```
Comment (jmarrec) [inline lib/measures/SetCOPforSingleSpeedDXCoolingUnits/measure.rb:272]: COP for CoilCoolingDX(Single|Two)Speed is now a double, not an OptionalDouble. To maintain backward compat, wrap it into an OptionalDouble anyways, it'll work before/after

File: lib/measures/add_zone_ventilation_design_flow_rate_object/measure.rb [hunk 3]
```diff
@@ -187,8 +187,11 @@ def run(model, runner, user_arguments)
     zone_ventilation = OpenStudio::Model::ZoneVentilationDesignFlowRate.new(model)
     zone_ventilation.addToThermalZone(zone)
     zone_ventilation.setVentilationType(vent_type)
-    zone_ventilation.setDesignFlowRateCalculationMethod('Flow/Zone')
     zone_ventilation.setDesignFlowRate(design_flow_rate_si)
+    if OpenStudio::VersionString.new(OpenStudio.openStudioVersion) < OpenStudio::VersionString.new('3.5.0')
+      # Design Flow Rate Calculation Method is automatically set in 3.5.0+
+      zone_ventilation.setDesignFlowRateCalculationMethod('Flow/Zone')
+    end
```
Comment (jmarrec) [inline lib/measures/add_zone_ventilation_design_flow_rate_object/measure.rb:194]: Another API change

File: lib/measures/generic_qaqc/resources/check_envelope_conductance.rb [hunk 4]
```diff
@@ -155,9 +155,13 @@ def check_envelope_conductance(category, target_standard, min_pass, max_pass, na
...
       std = Standard.build(target_standard)
-        
+
       # gather building type for summary
-      bt_cz = std.model_get_building_climate_zone_and_building_type(@model)
+      if Gem::Version.new(OpenstudioStandards::VERSION) > Gem::Version.new('0.2.16')
+        bt_cz = std.model_get_building_properties(@model)
+      else
+        bt_cz = std.model_get_building_climate_zone_and_building_type(@model)
+      end
```
Comment (jmarrec) [inline lib/measures/generic_qaqc/resources/check_envelope_conductance.rb:164]: Standards 0.2.17-rc1 renamed a method: `model_get_building_climate_zone_and_building_type` -> `model_get_building_properties`

cf commit https://github.com/NREL/openstudio-standards/commit/88db9d4d03db2d359ac94ab7fb721c5de887c5e4

**Comment by jmarrec** [comment]
@tijcolem there are similar changes needed on the openstudio-standards side, I opened https://github.com/NREL/openstudio-standards/pull/1393 with some of them

File: Gemfile [hunk 5]
```diff
@@ -15,3 +15,6 @@ if allow_local && File.exist?('../OpenStudio-extension-gem')
 elsif allow_local
   gem 'openstudio-extension', github: 'NREL/OpenStudio-extension-gem', branch: 'develop'
 end
+
+# TODO: temp
+gem 'openstudio-standards', github: 'NREL/openstudio-standards', branch: '3.5.0_changes'
```
Comment (jmarrec) [inline Gemfile:20]: * [ ] TODO: temp

**Review by DavidGoldwasser — APPROVED** [review]

**Pull request merged**